platforms/platforms_deprecated.go

```diff
@@ -0,0 +1,176 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package platforms
+
+import (
+	"github.com/containerd/platforms"
+	specs "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+// Platform is a type alias for convenience, so there is no need to import image-spec package everywhere.
+//
+// Deprecated: use [specs.Platform].
+type Platform = specs.Platform
+
+// DefaultSpec returns the current platform's default platform specification.
+//
+// Deprecated: use [platforms.DefaultSpec].
+func DefaultSpec() specs.Platform {
+	return platforms.DefaultSpec()
+}
+
+// Default returns the default matcher for the platform.
+//
+// Deprecated: use [platforms.Default].
+func Default() platforms.MatchComparer {
+	return platforms.Default()
+}
+
+// DefaultString returns the default string specifier for the platform.
+//
+// Deprecated: use [platforms.DefaultString].
+func DefaultString() string {
+	return platforms.DefaultString()
```

Comment:
Can you try to keep this as `return Format(DefaultSpec())`, platforms `DefaultString()` is not using `return FormatAll(DefaultSpec())` which includes the OS version.

Reply:
thx; can give that a try; I had a quick peek and it looks like that's only used in 1 place (in `ctr` to set the default value).

Perhaps that one should use the underlying code instead 🤔

Reply:
oh, never mind, looking in the wrong place (imgcrypt). Let me look further.

Reply:
I pushed a commit, but perhaps we should indeed consider to make this a separate function after all (new function to return default including os-version, and keep the old one without 🤔)